addons/fieldmanual/CfgHints.hpp

```diff
@@ -106,6 +106,14 @@ class CfgHints {
             description = CSTRING(Items_EarPlugs_Description);
             tip = "<a href='https://ace3.acemod.org/wiki/feature/hearing'>Hearing Wiki</a>";
         };
+        class ACE_EHP: ACE_FieldManual_Base {
+            logicalOrder = 10;
```

Comment:
Has the same `logicalOrder` as `ACE_EarPlugs`, which leads to weird behaviour: "Electronic Hearing Protection" is listed before "Earplugs" in the field manual on my end.
Means all the following `logicalOrder` need to be incremented by 1.
```suggestion
            logicalOrder = 11;
```